charts/kyverno/values.yaml

```
@@ -336,17 +328,30 @@ enforceSecurityContext:
   # - my-namespace
 
 autoInjectDockerPullSecrets:
-  enabled: true
+  enabled: false
+
+  # You can specify existing secrets of type "kubernetes.io/dockerconfigjson" under "existingSecretRef" to
+  # clone and use instead of letting Kyverno create these secrets from the chart values. Note that the
+  # secrets to copy must contain valid ".dockerconfigjson" data. Additionally, the existing secrets to
+  # be used must be specified under "existingImagePullSecrets" in the values.yaml.
+  # It is possible to use both existing secrets and have Kyverno create new secrets from the chart at the
+  # same time. For this, specify existing secrets under "existingSecretRef" and the ones to create under
+  # "secrets" as displayed in the examples.
+  existingSecretRef:
```

```suggestion
  existingSecretRef: []
```

Only a minor one I missed in the first place. When everything below is commented out, add an empty object of the correct type. In this case should be: []

Otherwise the data type is wrong in the [README.md](https://github.com/iits-consulting/charts/pull/181/files#diff-acf6b408c7bc3c66a16153d84b3a23e71c5ad99be73479c0386d8367713f54ecR24)